textbook/12/1/uniform.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<p>This is well-stated.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chicago-cdac/textbook-datascience-1/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>